save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/LnkUserOrganizationController.kt

```diff
@@ -109,16 +107,18 @@ class LnkUserOrganizationController(
     fun getRole(
         @PathVariable organizationName: String,
         @RequestParam(required = false) userName: String?,
-        authentication: Authentication,
-    ): Mono<Role> = getUserAndOrganizationWithPermissions(
-        userName ?: authentication.toUser().name!!,
-        organizationName,
-        Permission.READ,
-        authentication,
-    )
-        .map { (user, organization) ->
-            lnkUserOrganizationService.getRole(user, organization)
-        }
+        authentication: Authentication?,
+    ): Mono<Role> = authentication?.let {
+        getUserAndOrganizationWithPermissions(
+            userName ?: authentication.toUser().name!!,
+            organizationName,
+            Permission.READ,
+            authentication,
+        )
+            .map { (user, organization) ->
+                lnkUserOrganizationService.getRole(user, organization)
+            }
+    } ?: Role.NONE.toMono()
```

Comment:
Do we have role viewer?

Reply:
Viewer role for authorized users who are not members of this organization.